src/components/bookmarkScreen/bookmarkScreen.component.js

```diff
@@ -35,7 +25,7 @@ export const BookmarkScreenComponent = ({headerAnimatedStyle, renderBookmarkItem
         <FlatList
           data={bookmarks}
           bounces={true}
-          renderItem={ (item) => renderBookmarkItem(item) }
+          renderItem={ renderBookmarkItem }
```

Comment:
don't add spaces for props, renderItem={ renderBookmarkItem } -> renderItem={renderBookmarkItem}